### PR TITLE
Mobile version skeleton

### PR DESCRIPTION
**Pull request by Christabelle143**
### In this milestone I did the following:

- Add a header and a headline section in the index.html file which I created.
- Add stylings to the header and headline section in the style.css file.
- Modified and updated my README.md file using the template provided.
- Add the images folder with images downloaded from the first Figma template.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                ><img src="images/Vector2.png" alt="github icon"/></a>
+                <a href="https://linkedin.com/youtalactioChristabelle" target="blank"
+                ><img src="images/linkedinicon.png" alt="linkedin icon"/></a>
+                <a href="https://twitter.com/christabelleyo" target="blank"
+                ><img src="images/Vector.png" alt="meduim icon"/></a>
+                <a href="https://twitter.com/christabelleyo" target="blank"
+                ><img src="images/Vector1.png" alt="angellist icon"/></a>
+            </p>
+        </section>
+    </div>
```
Comment (BenMukebo) [inline index.html:49]: - [ ] let dive into your design now, please take a look at this image below compared to the original design.
   - your intro title is not using the same font and color as the original one
   - please adjust your padding and margin to make some few spaces around the `let connect` text.
   - right at the bottom left, please refer to the Figma template to do what is supposed to be there, you should crop that corner as shown with the red line.
   
b

- [ what it should look like ]
bela fig

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi ** @Christabelle **

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight 💪🏻

- No linter errors. ✔️ 
- Github flow was correctly used.  ✔️   
- Good commit messages ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_


- [ ] Kindly, Add a descriptive README file to your project as required please use this [template](https://github.com/microverseinc/readme-template) You can copy past but remember to customize it to your project.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Christabelle143** [comment]
Hi, **@BenMukebo** 
The image I got from the Figma template is a full one and I also find difficulties in cropping the picture to get the edges like the one you showed above. Don't know how to get it done

**Comment by BenMukebo** [comment]
Hello @Christabelle143, you can use border-radius on that specific corner

**Comment by Christabelle143** [comment]
Hi, ** @BenMukebo** 
I have tried the border-bottom-right-radius to no avail. It's not working. And I don't know why.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    <title>Mobile Version Skeleton</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body class="container">
+    <header>
+        <nav>
+            <a href="#" class="logo">
+                <h2>Crysty</h2>
+            </a>
```
Comment (thecodechaser) [inline index.html:17]: - [ ] As requested by the previous reviewer, Please remove the logo from the `<nav>` tag and just keep it in the header tag directly. The header is not part of our navbar so it is not good practice to place it inside `<nav>`
- [ ] Please remove `<h2>` inside anchor tag, It's not good practice to place headings inside an anchor tag, You can place the text directly inside the anchor tag.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+            <a href="#" class="menu"><img src="images/Union.png" alt="dropdown menu"> </a>
+        </nav>
+    </header>
+    <main class="profile">
+        <section class="infos">
+            <h1>I’m Christine Glad to see you!</h1>
+            <p>
+                I’m a software developer! I can help you build a product , feature or
+                website Look through some of my work and experience! If you like what
```
Comment (thecodechaser) [inline index.html:26]: - [ ] The background images are overflowing the content and the design is not looking good, Please fix the background images and make sure it matches the design on the [template ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)to make the project looks more professional.

### Your design

image

### Figma design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Christabelle143👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**